pkg/nsxt/config/config_ini_legacy_test.go

```diff
@@ -23,10 +23,14 @@ import (
 )
 
 func TestINIValidateTokenConfig(t *testing.T) {
-	cfg := &NsxtINI{
+	cfg := &NsxtINI{}
+	err := cfg.validateConfig()
+	assert.EqualError(t, err, "user or vmc access token or client cert file or secret for user must be set")
+
+	cfg = &NsxtINI{
```

Comment:
I think we can add a testcase where VMCAccessToken is empty but VMCAuthHost is provided

Reply:
OK, then we need add validation for this case as well :)